pages/02.install/06.truenas-scale/default.md

```diff
@@ -24,6 +24,8 @@ To access Kavita, go to the Installed Applications page and click Kavita's "Open
 
 ### Installing Kavita (Nightly)
 
+##### Note: Kavita's nightly release is meant for beta testers, and breaking changes may slip in. For the most stable experience, install Kavita (Stable).
```

Comment:
Make this ! Message. 

The ! Styles as an alert

Reply:
Fixed